src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
-'use strict';
+"use strict";
 
-// write your code here
+const div = document.createElement("div");
+
+const success = (result) => {
+  div.className = "message";
+  div.innerText = result;
+  document.body.append(div);
+};
+
+const error = (result) => {
+  div.className = "message error-message";
+  div.innerText = result;
+  document.body.append(div);
+};
+
+const promise1 = new Promise((resolve, reject) => {
```

Comment:
The promise1 does not have a reject condition. It is better to add conditions for both resolve and reject for the consistency and predictability of the code.